db/migrate/20181025230931_add_comments_to_vms_table.rb

```diff
@@ -0,0 +1,102 @@
+                                                     "are unreserved resources available. VMWare only."
+
+    change_column_comment :vms, :memory_shares, "The number of memory shares allocated. Used to determine resource allocation "\
+                                             "in case of resource contention. VMWare only."
+
+    change_column_comment :vms, :memory_shares_level, "Memory shares allocation level. VMWare only."
+
+    change_column_comment :vms, :miq_group_id, "MIQ Group associated with the VM."
+    change_column_comment :vms, :name, "The name of the VM/Template."
+    change_column_comment :vms, :orchestration_stack_id, "ID of the orchestration stack associated with the VM."
+    change_column_comment :vms, :power_state, "The current power state, typically 'on' or 'off'."
+    change_column_comment :vms, :previous_state, "The previous power state change."
+    change_column_comment :vms, :publicly_available, "Indicates whether the VM is public or private."
+    change_column_comment :vms, :raw_power_state, "The power state refreshed from the EMS."
+    change_column_comment :vms, :registered, "Indicates whether the VM registered to a Provider."
+    change_column_comment :vms, :resource_group_id, "ID of the resource group that the VM/Template belongs to."
+    change_column_comment :vms, :retired, "Indicates whether or not the VM is retired."
+    change_column_comment :vms, :retirement_last_warn, "Date of last retirement warning."
+    change_column_comment :vms, :retirement_requester, "Userid of the user who requested retirement."
+    change_column_comment :vms, :retirement_state, "The current retirement state."
+    change_column_comment :vms, :retirement_warn, "Warning message when VM was put into retirement."
+    change_column_comment :vms, :retires_on, "Timestamp the VM is to be retired."
+    change_column_comment :vms, :smart, "Indicates whether or not smart proxy is enabled. Deprecated."
+    change_column_comment :vms, :standby_action, "Action taken when VM is put into standby mode."
+    change_column_comment :vms, :storage_id, "ID of the storage object associated with the VM."
+    change_column_comment :vms, :storage_profile_id, "ID of the storage profile associated with the VM."
+    change_column_comment :vms, :state_changed_on, "The timestamp of the last power state change."
+    change_column_comment :vms, :template, "Indicates whether this is a VM or template."
+    change_column_comment :vms, :tenant_id, "ID of the tenant associated with the VM."
+    change_column_comment :vms, :tools_status, "Status of guest tools."
+    change_column_comment :vms, :type, "An internal class name that includes the provider type."
+    change_column_comment :vms, :uid_ems, "A globally unique reference for the VM across EMS."
+    change_column_comment :vms, :updated_on, "The last timestamp the VM was modified within the app."
+    change_column_comment :vms, :vendor, "The vendor for the VM, e.g. Azure, Amazon, etc."
```

Comment:
vendor values are lowercased, so the examples are incorrect. See https://github.com/ManageIQ/manageiq/blob/242bd55197020baf8c385dc35110051cc2947378/app/models/vm_or_template.rb#L40-L53